yara-x/src/modules/string.rs

```diff
@@ -0,0 +1,86 @@
+use crate::modules::prelude::*;
+use crate::modules::protos::string::*;
+
+#[module_main]
+fn main(_ctx: &ScanContext) -> String {
+    // Nothing to do, but we have to return our protobuf
+    let string_proto = String::new();
+    string_proto
+}
+
+#[module_export(name = "to_int")]
```

Comment:
Don't use `name = "to_int"` if the Rust function is already named `to_int`.

Reply:
Ok, i will resolve this.

Reply:
Fixed